src/components/Widget/index.js

```diff
@@ -398,6 +398,19 @@ class Widget extends Component {
           // Let's drop all the messages that we had
           dispatch(dropMessages());
 
+          // Retrieve the stored item from sessionStorage
+          const storedSession = sessionStorage.getItem(SESSION_NAME);
+
+          // Parse the stored item into an object
+          const object = JSON.parse(storedSession);
+
+          // Clear the conversation
+          object.conversation = [];
```

Comment:
You are explicitly using `sessionStorage` which may be empty if the widget is setup to use `localStorage`, therefore your `object` will be `null` and your `object.conversation = []` would end up with a `TypeError`.